app/views/shared/_commit_message_replace_container.html.haml

```diff
@@ -0,0 +1,14 @@
+.form-group.commit_message-group
+  = label_tag 'commit_message_replace', class: 'control-label' do
```

Comment:
@tsigo I checked another file _commit_message_container.html.haml in the newest repository, it also keep using with '**', so I keep the same, should I change?